tests/BuildCommandTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Umpirsky\ListGenerator;
+
+use Umpirsky\ListGenerator\Builder\Builder;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Tester\CommandTester;
+use Umpirsky\ListGenerator\Exporter\Iterator as ExportIterator;
+use PHPUnit\Framework\TestCase;
+
+class BuildCommandTest extends TestCase
+{
+    /**
+     * @var CommandTester
+     */
+    private $commandTester;
+
+    protected function setUp()
+    {
+        $builder = new Builder(new ImporterInstance(), 'importer_path');
+        $command = $builder->get('build');
+
+        $this->commandTester = new CommandTester($command);
+    }
+
+    protected function tearDown()
+    {
+        $exportIterator = new ExportIterator();
+        foreach ($exportIterator as $exporter) {
+            $importerFile = glob(__DIR__.'/../importer_path/en/*.'.$exporter->getFormat());
+            foreach($importerFile as $file) {
+                file_exists($file) ? unlink($file) : false;
+            }
+        }
+
+        rmdir(__DIR__.'/../importer_path/en');
+        rmdir(__DIR__.'/../importer_path');
+    }
+
+    /**
+     * @dataProvider argumentDataProvider
+     */
+    public function testBuildCommand($language, $format)
+    {
+        $this->commandTester->execute([
+            'language' => $language,
+            'format' => $format,
+        ]);
+
+        $this->assertSame(0, $this->commandTester->getStatusCode());
+        $this->assertContains('', $this->commandTester->getDisplay());
+    }
+
+    public function argumentDataProvider()
+    {
+        return [
+            ['en', 'json'],
```

Comment:
You can use `Umpirsky\ListGenerator\Exporter\Iterator` here.

Reply:
I don't use the ```Umpirsky\ListGenerator\Exporter\Iterator```  here because the ```Iterator``` will contain the realted ```sql``` format.
But I will not write ```sql``` format tests at this time.
Perhaps I will do this in next PR.